Python/cp/stock_span.py

```diff
@@ -0,0 +1,42 @@
+''' Stock Span Problem
+Given a list of prices of a stock for N number of days, find stock span for each day.
+Stock span is defined as a number of consecutive days prior to the current day when the price of a stock was less than or equal to the price at current day.
+'''
+
+def calcSpan(price):
+    # Create an empty stack
```

Comment:
remove comments from middle of the code

Reply:
Should I remove all the comments from middle of the code? @plazzy99

Reply:
yes

Reply:
done